chatkit-core/src/main/kotlin/com/pusher/chatkit/cursors/CursorSubscriptionEvent.kt

```diff
@@ -1,12 +1,13 @@
 package com.pusher.chatkit.cursors
 
+import com.pusher.chatkit.cursors.api.CursorApiType
 import elements.Error
 
-sealed class CursorSubscriptionEvent {
-    data class OnCursorSet(val cursor: Cursor) : CursorSubscriptionEvent()
-    data class InitialState(val cursors: List<Cursor>) : CursorSubscriptionEvent()
+internal sealed class CursorSubscriptionEvent {
+    data class OnCursorSet(val cursor: CursorApiType) : CursorSubscriptionEvent()
+    data class InitialState(val cursors: List<CursorApiType>) : CursorSubscriptionEvent()
     data class OnError(val error: Error) : CursorSubscriptionEvent()
```

Comment:
this shall be removed, and the logic in the parser logic that leads to creation of those objects. if backend API gets extended it shouldn't be treated nor moreover signalled on our side as an error but be just ignored

Reply:
Sorry, which part should be removed? GH didn't highlight what you selected perhaps.

Reply:
`data class OnError(val error: Error) : CursorSubscriptionEvent()` can be removed, also from the parser

Reply:
👍 have removed, and refactored the OnError to be like the PresenceSubscriptionEventParser.